actors/common/src/main/java/org/sunbird/learner/actors/BackgroundJobManager.java

```diff
@@ -628,4 +631,13 @@ private void updateUserNotesToEs(Request actorMessage) {
         (String) noteMap.get(JsonKey.ID),
         noteMap);
   }
+
+  private void mergeUserDetailsToEs(Request mergeRequest) {
+    String mergeeId = (String) mergeRequest.get("mergeeId");
```

Comment:
use same termonologu either user fromAccount and toAccount or mergeeId or mergerId

Reply:
followed fromAccountId and toAccountId while getting from request.